### ISSUE TITLE
Could you make same thing for goldshell Al box 2 :) ?

### ISSUE THREAD

**Issue by fodger** [rdugan/iceriver-alph-oc#1]
it could be wonderful ...

**Comment by RainBitcoins** [comment]
Not usually his thing, but ya would be nice

**Comment by rdugan** [comment]
@fodger I know nothing about these asics.  I would need access to one to even consider it.  Are you even able to ssh into them?

**Comment by RainBitcoins** [comment]
> @fodger I know nothing about these asics. I would need access to one to even consider it. Are you even able to ssh into them?

Have you got your hands on an RX0 yet? I'm wondering why these things hash at such a low rate? I'm wondering if they were just trying to keep it at that 100 watts mark. If it's the case than there should be plenty of headroom to play with.

**Comment by rdugan** [comment]
@RainBitcoins yeah, working on it.  i'm assuming it's just different hashrates because of different algos. form factor and chip count are the same as the other 0 models, so i wouldn't expect h/r will always be the same across algos

**Comment by RainBitcoins** [comment]
> @RainBitcoins yeah, working on it. i'm assuming it's just different hashrates because of different algos. form factor and chip count are the same as the other 0 models, so i wouldn't expect h/r will always be the same across algos

Nice.... Iceriver did tell me they only made 900 units and they couldn't tell me if they were making anymore as I had told them I wanted to buy more.
